### DISCUSSION TITLE
Welcome to FlipIt Discussions!

### DISCUSSION THREAD

**Discussion by phaselden** [phaselden/FlipIt#14] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by clutch-man** [comment]
I have to say, good job.
support 24h and size zoom...
hope to add page turning style

**Comment by JWSun375** [comment]
page-turning style
Plz ！！

**Comment by Ridbowt** [comment]
Great screensaver! I had problems with Fliqlo - as far as I can remember, it just stopped working - but FlipIt works without problems + it's faster than the above one!
